ckanext/datawagovautheme/templates/home/snippets/search.html

```diff
@@ -15,7 +15,10 @@ <h3 class="heading">{{ _("Search data") }}</h3>
   <div class="tags">
     <h3>{{ _('Popular tags') }}</h3>
     {% for tag in tags %}
-      <a class="tag" href="{% url_for controller='package', action='search', tags=tag.name %}">{{ h.truncate(tag.display_name, 22) }}</a>
+      <a class="tag" href="{% url_for controller='package', action='search', tags=tag.name %}">
+      {# {{ h.truncate(tag.display_name, 22) }} #}
```

Comment:
No need to include the commented our version of the old code here - if we need it we'll check git history.